src/controller/userController.js

```diff
@@ -49,13 +50,54 @@ exports.updatedata = async (req, res) => {
 };
 
 //  Delete data
-
 exports.deletedata = async (req, res) => {
   const userId = req.params.userId;
   try {
-    await model.findByIdAndDelete(userId);
+    await userModel.findByIdAndDelete(userId);
     res.status(200).send("data deleted sucessfully");
   } catch (err) {
     res.status(400).send(err.message + "Data Deletion Unsucessufl");
   }
 };
+
+// Login cheak
+exports.login = async (req, res) => {
+  const { userName, password } = req.body;
```

Comment:
we should use `email` instead of userName

Reply:
and add `unique: true` validation for email

Reply:
done